src/schedule/schedule_inner.rs

```diff
@@ -148,3 +150,19 @@ impl Schedule {
 		}
 	}
 }
+
+/// Get schedule JSON from definition file
+pub fn get_schedule_from_config() -> ScheduleDefinition {
+	if !fs::metadata("./def.json").is_ok() {
```

Comment:
I would recommend using `Path::new("./def.json").exists()` rather than checking a file's metadata, only to immediately discard it. If you choose to keep it the way it is, please use `.is_err()` rather than `!.is_ok()`, this is an anti-pattern and emits a clippy warning.